stories/__tests__/SQFormDatePicker.stories.test.tsx

```diff
@@ -96,8 +96,8 @@ describe('SQFormDatePicker Tests', () => {
     expect(calendarDialog).toBeInTheDocument();
     expect(calendarDialog).toBeVisible();
 
-    const dateOptions = within(calendarDialog).getAllByRole('cell');
-    const selectedDate = dateOptions[0];
+    const dateOptions = within(calendarDialog).getAllByRole('button');
+    const selectedDate = dateOptions[4];
```

Comment:
I had issues with this story not passing for me.  We needed to actually get all of the buttons within the `calendarDialog`.  The 1st of the month corresponds to the fifth button inside the `calendarDialog`.  A bit hacky, but the best I could figure out.  I can revisit if needed, but don't think that it is very likely we would be adding/removing buttons that would cause this hard-coded value not to work.

Reply:
I think this will work for now. I already have a line item to go update the tests for this component once we upgrade to version 5.
https://github.com/SelectQuoteLabs/SQForm/issues/358